### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PeterDaveHello**
**Pull Request Type**
Please select what type of pull request this is:
- [ ] Bugfix
- [ ] Feature Implementation
- [x] Documentation

**Description**

Add link of LibRedirect and remove the download links of Privacy Redirect.

The development of Privacy Redirect looks pending for a while, the forked LibRedirect seems to be its successor with more active maintenance, should be worthy to be mentioned here.

### CODE REVIEW

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Dont change the order

Prvacyresirecr first
Libredirect second

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — CHANGES_REQUESTED** [review]
See previous comment

**Comment by PeterDaveHello** [comment]
Updated.

File: README.md [hunk 1]
```diff
@@ -14,10 +14,12 @@ addressed.
 
 ### Browser Extension
 
-FreeTube is supported by the [Privacy Redirect](https://github.com/SimonBrazell/privacy-redirect) extension, which will allow you to open YouTube links into FreeTube. You must enable the option within the advanced settings for it to work.
+FreeTube is supported by the [Privacy Redirect](https://github.com/SimonBrazell/privacy-redirect) and [LibRedirect](https://github.com/libredirect/libredirect) extension, which will allow you to open YouTube links into FreeTube. You must enable the option within the advanced settings for it to work.
 
 Download Privacy Redirect for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/privacy-redirect/) or [Google Chrome](https://chrome.google.com/webstore/detail/privacy-redirect/pmcmeagblkinmogikoikkdjiligflglb).
 
+Download Privacy Redirect for [Firefox](https://addons.mozilla.org/firefox/addon/libredirect/) or [Google Chrome](https://github.com/libredirect/libredirect/blob/master/chromium.md).
```
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [inline README.md:21]: It should say Download LibRedirect

The chrome link is not an url from the chrome store

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — CHANGES_REQUESTED** [review]

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Review by ChunkyProgrammer — APPROVED** [review]

**Review by absidue — APPROVED** [review]

**Pull request merged**